interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -289,7 +289,7 @@
     <title><?php echo xlt('Rx Dispensed History'); ?></title>
     <head>
 
-        <?php Header::setupHeader([ 'opener', 'jquery-ui', 'jquery-ui-redmond', 'pure', 'jscolor' ]); ?>
+        <?php Header::setupHeader(['opener', 'pure', 'jscolor']); ?>
 
         <link rel="stylesheet" href="../../forms/<?php echo $form_folder; ?>/css/style.css" type="text/css">
```

Comment:
Relative paths issue here -- should be using root globals variable to prevent pathing issues. It's either gonna be $GLOBALS['rootdir'] or $include_root @bradymiller which one is it